src/Chapter1.hs

```diff
@@ -631,9 +645,11 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
-
+sumLast2 :: Int -> Int
+sumLast2 n = 
+    let lastD = lastDigit n
+        b4Last = div (mod n 100) 10
```

Comment:
Oh cool, reusage of the `lastDigit` function 💡 
Note that here you also would get the wrong number on the negative input. You can use `abs` function to avoid it 🙂